thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/DataService.js

```diff
@@ -146,7 +146,7 @@ DataService.prototype = {
     fetchAnomaliesForMetricIds : function(startTime, endTime, pageNumber, metricIds, functionName, filterOnly, callback, spinner) {
       const url = constants.SEARCH_ANOMALIES_METRICIDS + startTime + this.URL_SEPARATOR + endTime + this.URL_SEPARATOR + pageNumber;
       const data = {
-        metricIds: JSON.stringify(metricIds),
+        metricIds: metricIds,
```

Comment:
is JSON.stringify adding quotes around metricIds? how about passing a list of metricIds? Backend would expect passing metricId1,metricId2,metricId3

Have we recently changed this part from frontend? Since searching by metricIds was working.